### DISCUSSION TITLE
Görkem Arslan for substantial code, code review, and software architecture contributions

### DISCUSSION THREAD

**Discussion by brylie** [CiviWiki/OpenCiviWiki#1068] [Nominations]
I nominate  Görkem Arslan (@gorkemarslan) to join the CiviWiki Core development team based on their significant contributions to code, code quality, pull request review, and project architecture.

Would you please express your support by upvoting this comment?

**Comment by gorkemarslan** [comment]
I didn't expect that. It's really a surprise for me. This is the first time that I am nominated for something. This is such an honor for me!

@brylie, I think you deserve the biggest thanks here. It is not easy to manage such a long-lasting project and keep it updated. You're really patient and helpful. You really know how to motivate us.

We all learn from each other. 
Guys, you and the project contribute to me a lot.

 ✨ Thanks for everything. ✨

**Reply by AbhijithGanesh** [reply]
I second this nomination, Gorkem Arslan was an amazing contributor and should be added.

**Reply by AbhijithGanesh** [reply]
When will he be added to the organization?

**Reply by brylie** [reply]
Let's give the nomination process some period for review. It's only been five days currently.

**Comment by brylie** [comment]
As a follow-up to this nomination, today, I've added @gorkemarslan as a project maintainer (i.e., a core member). :smiley:

**Comment by gorkemarslan** [comment]
I feel honored for this role. Thank you everyone, thank you our community! :smile:

**Reply by AbhijithGanesh** [reply]
Congratulaions📣📣📣 I am glad that I have worked alongside you contributing
